routes/api/contacts.js

```diff
@@ -11,6 +11,7 @@ const {
 	addContactsSchema,
 	contactUpdateStatusValidationShema,
 } = require('../../models/validator');
+const auth = require('../../auth/auth');
```

Comment:
Hej, ten middleware nei jest używany, sugerowałbym dodać ten middleware do ścieżek w "contacts". Dostęp do API contacts powinien być możliwy tylko dla zalogowanych użytkowników (z tokenem)

Reply:
Poprawione, przyznam że kompletnie zapomniałem bo miałem to pózniej zrobić bo borykałem się problemem.